android/src/main/java/com/thebluealliance/androidclient/viewmodels/ViewModelRenderer.java

```diff
@@ -0,0 +1,11 @@
+package com.thebluealliance.androidclient.viewmodels;
+
+import android.content.Context;
+import android.support.annotation.Nullable;
+import android.support.annotation.WorkerThread;
+
+public interface ViewModelRenderer<VIEWMODEL, ARGS> {
+
+    @WorkerThread
+    @Nullable VIEWMODEL renderToViewModel(Context context, @Nullable ARGS args);
```

Comment:
Gotcha. I was suggesting using a parent class not for any real generic reason, but to make it obvious that all the `ViewModel` classes share something (and that we have a place to put common code in the future, should we need it) so we don't need the `List<Object>` type like above (instead use `List<ViewModel>`, which make it much more obvious to the casual reader what is contained in the list, even though it doesn't add any functional value). Just a thought